recipes/simdjson/0.4.6/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class SimdjsonConan(ConanFile):
+    name = "simdjson"
+    version = '0.4.6'
+    description = "Parsing gigabytes of JSON per second"
+    topics = ("conan", "json", "parser", "simd", "format")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lemire/simdjson"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "threads": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True,
+                       'threads': True}
+    _source_subfolder = "source_subfolder"
+
+    @property
+    def _supported_cppstd(self):
+        return ["11", "gnu11","14", "gnu14", "17", "gnu17", "20", "gnu20"]
+
+    def _has_support_for_cpp11(self):
+        supported_compilers = [("apple-clang", 10), ("clang", 6), ("gcc", 7), ("Visual Studio", 15.7)]
```

Comment:
c++11 is supported on lower compiler versions, the combination seems odd :thinking: Would in be possible to get some insight I'm curious?

Si je retroune au universite, ce sera pour une de tes cours!